soh/soh/Enhancements/debugger/debugSaveEditor.cpp

```diff
@@ -377,21 +401,92 @@ void DrawInfoTab() {
-            gSaveContext.linkAge = 0;
+    ImGui::InputScalar("Entrance Index", ImGuiDataType_S32, &gSaveContext.entranceIndex);
+    InsertHelpHoverText("From which entrance did Link arrive?");
+    ImGui::InputScalar("Cutscene Index", ImGuiDataType_S32, &gSaveContext.cutsceneIndex);
+    InsertHelpHoverText("Which cutscene is this?");
+    ImGui::InputScalar("Navi Timer", ImGuiDataType_U16, &gSaveContext.naviTimer);
+    InsertHelpHoverText("Navi wants to talk at 600 units, decides not to at 3000.");
+    ImGui::InputScalar("Timer 1 State", ImGuiDataType_S16, &gSaveContext.timer1State);
+    InsertHelpHoverText("Heat timer, race timer, etc. Has white font");
+    ImGui::InputScalar("Timer 1 Value", ImGuiDataType_S16, &gSaveContext.timer1Value, &one, NULL);
+    InsertHelpHoverText("Time, in seconds");
+    ImGui::InputScalar("Timer 2 State", ImGuiDataType_S16, &gSaveContext.timer2State);
+    InsertHelpHoverText("Trade timer, Ganon collapse timer, etc. Has yellow font");
+    ImGui::InputScalar("Timer 2 Value", ImGuiDataType_S16, &gSaveContext.timer2Value, &one, NULL);
+    InsertHelpHoverText("Time, in seconds");
+     
+    std::string audioName;
+    switch (gSaveContext.audioSetting) { 
+        case 0:
+            audioName = "Stereo";
+            break;
+        case 1:
+            audioName = "Mono";
+            break;
+        case 2:
+            audioName = "Headset";
+            break;
+        case 3:
+            audioName = "Surround";
+            break;
+        default:
+            audioName = "?";
+    }
+    if (ImGui::BeginCombo("Audio", audioName.c_str())) {
+        if (ImGui::Selectable("Stereo")) {
+            gSaveContext.audioSetting = 0;
         }
-        if (ImGui::Selectable("Child")) {
-            gSaveContext.linkAge = 1;
+        if (ImGui::Selectable("Mono")) {
+            gSaveContext.audioSetting = 1;
+        }
+        if (ImGui::Selectable("Headset")) {
+            gSaveContext.audioSetting = 2;
+        }
+        if (ImGui::Selectable("Surround")) {
+            gSaveContext.audioSetting = 3;
         }
 
         ImGui::EndCombo();
     }
-    */
+    InsertHelpHoverText("Sound setting");
+    
+    bool n64DDFlag = gSaveContext.n64ddFlag != 0;
+    if (ImGui::Checkbox("64 DD file?", &n64DDFlag)) {
+        gSaveContext.n64ddFlag = n64DDFlag;
+    }
+    InsertHelpHoverText("WARNING! If you save, your file may be locked! Use caution!");
+    
+    if (ImGui::BeginCombo("Z Target Mode", gSaveContext.zTargetSetting ? "Hold" : "Switch")) {
+        if (ImGui::Selectable("Switch")) {
+            gSaveContext.zTargetSetting = 0;
+        }
+        if (ImGui::Selectable("Hold")) {
+            gSaveContext.zTargetSetting = 1;
+        }
+        ImGui::EndCombo();
+    }
+    InsertHelpHoverText("Z-Targeting behavior");
+
 
+    ImGui::SetNextItemWidth(ImGui::GetFontSize() * 15);
+    std::vector<std::string> minigameHS = { "Horseback Archery", "Big Poe Points",
```

Comment:
This should be a `std::array<const char*>` since we dont need the features of either `std::vector` or `std::string`